### DISCUSSION TITLE
New instance best domain shortest or sub-dom

### DISCUSSION THREAD

**Discussion by jagsa** [mastodon/mastodon#25632] [General]
I'm thinking about standing up a Mas. server due to the changes at Twitter after 15 years its time for something new.

I have been hosting on the edge since IBM OS2 Bul. Board SMTP days in mid. '90s and was part of the team that managed the most popular southern hemi. web site so have exp. / servers I can use 'locally' to get this happening relatively easily.

From what I have read I need to be thinking carefully about the dom. 

Is it better to have a short dom eg. http://in.au or should I use a sub-dom eg. http://chat.in.au ?

I was also thinking about having a different instance for each state Qld.in.au or SA.in.au ? 

TIA

**Comment by mjankowski** [comment]
I think the answer here from a project perspective is ... it doesnt really matter? Either will work perfectly fine.

To the extent you have feelings about domain name length and the potential embrace/rejection of those from potential users, please listen to those intuitions.
